modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -160,7 +160,7 @@ This configuration does not enable serial console access on machines with a grap
 ====
 
 ifndef::only-pxe[]
-** For iPXE:
+** For iPXE (`x86_64` + `arm64`):
 +
 ----
 kernel http://<HTTP_server>/rhcos-<version>-live-kernel-<architecture> initrd=main coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-live-rootfs.<architecture>.img coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
```

Comment:
Hello @kelbrown20, I'd add another note at the end of this paragraph (just under the graphic console one) like:


*Note*: To network boot the CoreOS kernel on aarch64 architecture by iPXE, you need to use a version of iPXE built with the `IMAGE_GZIP` option enabled. See [https://ipxe.org/buildcfg/image_gzip](IMAGE_GZIP option in iPXE)

cc: @jeffdyoung

Reply:
Refers coreos/fedora-coreos-tracker#1019